core/lib/globals_shared.h

```diff
@@ -1838,6 +1838,8 @@ typedef union _dr_zmm_t {
     reg_t reg[IF_X64_ELSE(8, 16)]; /**< Representation as 8 or 16 registers. */
 } dr_zmm_t;
 
+typedef uint64 dr_opmask_t;
```

Comment:
Can this get some documentation (just right above this line) about how one should use this type?

Not sure if this is a good suggestion, but: looks like the current usage always uses the type as an array of size MCXT_NUM_OPMASK_SLOTS; should we make this typedef equal to the array type, or are we going to interact with the values as individual uint64 values at some point?

Reply:
I've added a comment. This is just the exposed register type as mentioned in the release docs.